.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "files.insertFinalNewline": true
+}
```

Comment:
You can do this in vscode setting instead so its applied to all projects

Reply:
Ah yeah I did do it in vscode but it looks like it automatically added this line to .vscode for this project specifically? Unless I thought I did it globally in vscode but didn't actually?